code-challenges.test.js

```diff
@@ -14,28 +14,132 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// Pseudocode:
+  // input1: fibLength1
+  // output1: [1, 1, 2, 3, 5, 8]
+
+  // Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+    // one parameter: number
+  // I googled the fibonacci sequence algorithm because I had no idea what it was. 
+  // I then adjusted the algorithm in order to apply it to my code. 
+  // (pseudocode continues on line 60)
+
+
 const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+Expected output: [1, 1, 2, 3, 5, 8]
+
+describe("fibSequence", () => {
+    it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.", () => {
+      let input1 = fibLength1
+      let result1 = [1, 1, 2, 3, 5, 8]
+      expect(fibSequence(input1)).toEqual(result1)
+    })
+  })
+
+  // Good fail!
+
 
 const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+// input2: fibLength2
+// output2: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+describe("fibSequence", () => {
```

Comment:
No need to separate this into 2 different tests.  Since this test is ultimately evaluating the fibSequence function, you can just provide multiple expect statements instead of a whole new test.